wip/SQLiteDriver.php

```diff
@@ -0,0 +1,533 @@
+<?php // phpcs:disable WordPress.Files.FileName.InvalidClassFileName,Generic.Files.OneObjectStructurePerFile.MultipleFound
```

Comment:
I'm assuming there's not much to review in this file yet so I'll skip over it. LMK if you wanted me to review this one as well.